src/lib/Cursor/index.ts

```diff
@@ -87,6 +87,19 @@ export class Cursor {
         cursor.x *= xRatio
         cursor.y *= yRatio
 
+        for (let x = cursor.x - 10; x <= cursor.x + 10; x++) {
```

Comment:
interessante abstrair essa funcionalidade me uma função, para ficar mais identificar o que esses for's fazem